tests/myapp/models.py

```diff
@@ -18,9 +18,6 @@ def custom_method(self):
 
 class CustomTreeManager(TreeManager):
 
-    def get_query_set(self):
-        return CustomTreeQueryset(model=self.model, using=self._db)
-
     def get_queryset(self):
         # Django 1.8 removed the fallbacks here.
```

Comment:
This comment is now irrelevant. Can you remove it as well?

Reply:
Done; I also removed the method `get_empty_query_set` which is no longer needed.